src/Dommel/DommelMapper.Insert.cs

```diff
@@ -1,10 +1,10 @@
+using Dapper;
```

Comment:
Leave this using at the bottom.

Reply:
Done. Visual Studio automatically sort `Using` statements for me upon saving.

Reply:
@DamienLaw you can configure this behavior in Visual Studio Options -> Text Editor -> C# -> Advances -> Place 'System' directives first when sorting usings.